hera_sim/visibilities/vis_cpu.py

```diff
@@ -45,22 +45,35 @@ class VisCPU(VisibilitySimulator):
         Whether to use the GPU version of vis_cpu or not. Default: False.
     mpi_comm : MPI communicator
         MPI communicator, for parallelization.
-    **kwargs
-        Passed through to :class:`~.simulators.VisibilitySimulator`.
+    ref_time
+        A reference time for computing adjustments to the co-ordinate transforms using
+        astropy. For best fidelity, set this to a mid-point of your observation times.
+        By default, if `correct_source_positions` is True, will use the first
+        observation time.
+    correct_source_positions
+        Whether to correct the source positions using astropy and the reference time.
+        Default is True if `ref_time` is given otherwise False.
     """
 
+    conjugation_convention = "ant1<ant2"
+    time_ordering = "time"
+
+    diffuse_ability = False
+    __version__ = __version__
+
     def __init__(
         self,
-        bm_pix=100,
-        use_pixel_beams=True,
-        polarized=False,
-        precision=1,
-        use_gpu=False,
+        bm_pix: int = 100,
+        use_pixel_beams: bool = True,
+        polarized: bool = False,
+        precision: int = 1,
+        use_gpu: bool = False,
         mpi_comm=None,
-        **kwargs
+        ref_time: Optional[Union[str, Time]] = None,
+        correct_source_positions: bool | None = None,
```

Comment:
`X | Y` typing is Python 3.10 syntax. The current setup config support `3.6`, `3.7`, and `3.8`

Reply:
See last line at https://docs.python.org/3.10/library/typing.html#typing.Union

Reply:
It might be more clear to just do `correct_source_positions: bool = False`

Reply:
Is this still need fixing?